workflows/geometry_workflows.md

```diff
@@ -40,11 +40,19 @@ argument. The double loaded corridor leads to space division and thermal zoning
 sliced core and perimeter zoning. It creates a circulation space type running down the center of the
 bar, surrounded by spaces of another space type. For an example, for a primary school the corridor will be paired with the classrooms.
 
-This workflow does not take the actual building footprint into account and does not consider
-the impact of self and context shading.
+![create_bar measure workflow diagram a](../doc_files/create_bar.png)
 
-![urbanopt measure workflow diagram](../doc_files/create_bar.png)
+As shown below workflow does take the actual building footprint into account by creating a second bar when necessary too match the perimeter by orientation non-rectangular geoJSON footprints. Additionally, custom high spaces like school cafeterias are pulled into their own bar from the rest of the building types. This more accurately represents the volume of those spaces. This workflow does not take into account the impact of self and context shading.
```

Comment:
- typo: "too" match (should be to)
- also "by orientation"? sounds  strange?
- should it be "from the rest of the building spaces"?